src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheckTest.java

```diff
@@ -65,11 +65,11 @@ public void testSpecified()
         final String pattern = "^a[A-Z][a-zA-Z0-9]*$";
 
         final String[] expected = {
-            "71:19: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat1", pattern),
-            "71:34: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat2", pattern),
-            "72:25: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat3", pattern),
+            "77:19: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat1", pattern),
+            "77:34: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat2", pattern),
+            "78:25: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat3", pattern),
         };
-        verify(checkConfig, getPath("InputParameterName.java"), expected);
+        verify(checkConfig, getPath("InputParameterName_1.java"), expected);
```

Comment:
Lets avoid `_` character as it does not satisfy java class naming conventions, you can use `InputParameterName1` or `InputParameterNameOne` or name it according to the test case `InputParameterNameSpecified`

Reply:
@timurt done